src/github.com/getlantern/flashlight/client/handler.go

```diff
@@ -78,11 +78,10 @@ func (client *Client) intercept(resp http.ResponseWriter, req *http.Request) {
 
 	// Respond OK as soon as possible, even if we don't have the outbound connection
 	// established yet, to avoid timeouts on the client application
-	success := make(chan bool, 1)
+	success := make(chan bool)
```

Comment:
I made this channel with a size 1 so the sender doesn't block. While now it won't really make a difference, if we never read the `success` channel, the goroutine will block and leak, ending up with tons of them. It is just a safety measure in case there is a reason for the channel never to be read, like a failure.
As I mentioned, this is not happening in this code, but I thought it was a good insurance to put in place.
Why would it be better to make it unbuffered?